src/Header/MobileMenu.tsx

```diff
@@ -0,0 +1,102 @@
+import { useState } from "react";
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import listIcon from './list.svg';
+import listCloseIcon from './listClose.svg';
+
+interface MobileMenuProps {
+  links: string[];
+}
+
+interface MenuButtonProps {
+  getOpen: () => boolean;
+  handleToggle: () => void;
+}
+
+interface MenuListProps {
+  links: string[];
+  getOpen: () => boolean;
+  closeList: () => void;
+}
+
+interface StyledListProps {
+  isOpen: boolean;
+}
+
+const MobileMenu = ({ links }: MobileMenuProps) => {
+  const [open, setOpen] = useState(false);
+
+  const getOpen = () => open;
+  const handleToggle = () => {
+    console.log(open);
+    setOpen(!open);
+  }
+  return (
+    <div>
+      <MenuButton getOpen={getOpen} handleToggle={handleToggle}/>
+      <MenuList links={links} getOpen={getOpen} closeList={handleToggle} />
+    </div>
+  );
+};
+
+const MenuButton = ({getOpen, handleToggle}: MenuButtonProps) => {
+  return (
+    <StyledMenuButton >
```

Comment:
I really like the way you styled this component, but I think you should rename `StyledMenuButton` because it's a little misleading. Also, I think using a className or an id to specify the button in the css in line 69 is a better practice (although this does work)

Reply:
Used className to specify the button in css. Not sure what I should call the button otherwise -- does `StyledHeaderListButton` work? Sounds a bit long though.

Reply:
I think it's fine, it doesn't really matter imo